specs/linux/containerd_container_events.table

```diff
@@ -0,0 +1,19 @@
+table_name("containerd_container_events")
+description("Track containerd container events.")
+schema([
+    Column("container_id", TEXT, "Container ID"),
+    Column("time", BIGINT, "When the event has been fired by the event framework"),
+    Column("timestamp", TEXT, "Time of when the event happened"),
+    Column("namespace", TEXT, "Namespace where the container is in"),
+    Column("topic", TEXT, "Specific type of the event"),
```

Comment:
enumerate possible values?

Reply:
Documented values based on reading source code for containerd, hopefully I got everything